Cargo.toml

```diff
@@ -36,6 +37,7 @@ with_ctap1 = ["crypto/with_ctap1"]
 with_nfc = ["libtock_drivers/with_nfc"]
 vendor_hid = ["libtock_drivers/vendor_hid"]
 fuzz = ["arbitrary", "std"]
+with_ed25519 = ["ed25519-dalek"]
```

Comment:
The "with_*" naming is discouraged and something we should migrate away from. Simply `ed25519` is good enough!

Reply:
Changed "with_ed25519" to "ed25519"
Regarding `nightly-2021-10-21`, I haven't checked newer versions, took first one after Rust 2021
Which versions should I try?